content/docs/1.7.0/nodes-and-volumes/nodes/node-conditions.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Node Conditions
+weight: 7
+---
+
+Node conditions describe the status of all worker nodes and are used to check the environment settings of worker nodes to identify potential issues before any system impact.
+
+Node conditions:
+
+- `Ready`:  
+  Indicates that the node is ready for Longhorn operations, including that a `longhorn-manager` pod is running on this node, the Kubernetes node is ready, and there is no physical resources pressure.  
+
+- `Schedulable`:  
+  Indicated that the node is not cordoned and workload can be scheduled to this node.
+
+- `MountPropagation`:  
+  Indicates that the node supports mount propagation. This is necessary for sharing of volumes mounted by a container with other containers in the same Longhorn pod, or to other Longhorn pods on the same node.  
+
+- `Multipathd`:  
+  Confirms if the `multipathd` service is not running on the node, which may affect the pod with the volume startup. See [Troubleshooting: `MountVolume.SetUp failed for volume` due to multipathd on the node](../../../../../kb/troubleshooting-volume-with-multipath).  
+
+- `RequiredPackages`:  
+  Checks if all required packages ([NFS client](../../../deploy/install/#installing-nfsv4-client), [iSCSI tool](../../../deploy/install/#installing-open-iscsi), [cryptsetup](../../../deploy/install/#root-and-privileged-permission), [dmsetup](../../../deploy/install/#using-the-environment-check-script)) exist for Longhorn  
```

Comment:
I just realized we don't have docs for the installations of dmsetup and cryptsetup. Can you add them as `open-iscsi`?